nmigen_stdio/test/test_spiflash.py

```diff
@@ -0,0 +1,155 @@
+import unittest
+from nmigen import *
+from nmigen.lib.fifo import SyncFIFO
+from nmigen.back.pysim import *
+
+from ..spiflash import *
+
+
+def simulation_test(dut, process):
```

Comment:
There may be an option to look at the VCD manually, but the test should be ideally self-checking.